# Define the goal
float64[4] ptu_joint_angels
---
# Define the result
int32 result
int32 FAILED=0
int32 SUCCEEDED=1
string info
---
# Define a feedback message